login.js

```diff
@@ -0,0 +1,65 @@
+
+function errorBoxOnOff(type, target) {
+  const targetNode = document.querySelector(`#${target}`)
+  targetNode.style.display = type === 'off' ? 'none' : 'block'
+}
+
+function focusOutCheck(event){
+  const value = event.target.value;
+  const id = event.target.id
+  if(id === 'email'){
+    if(value === ""){
+      errorBoxOnOff('on', `${event.target.id}ErrorBox`)
+      document.querySelector(`#${event.target.id}ErrorBox`).textContent = ' 이메일을 입력해주세요.'
```

Comment:
```suggestion
      document.querySelector(`#${id}ErrorBox`).textContent = ' 이메일을 입력해주세요.'
```

상단에 `id`를 선언해두셔서 이런식으로 재사용 가능합니다!